_posts/0000-01-02-greg707.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Before you marry a person, you should first make them use a computer with slow Internet to see who they really are." *Will Ferrell*

Reply:
I hope it goes on line 5.